### ISSUE TITLE
Screen reader does not announce the name of 'Next' and 'Previous' controls in the pagination section in windows.

### ISSUE THREAD

**Issue by msftedad** [ng-bootstrap/ng-bootstrap#4892]
**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26220.7523)
Browser: Edge (Version 144.0.3719.35)
Screen Reader: NVDA (2025.3.2)
URL: https://angular-ui.github.io/bootstrap/#!#getting_started:~:text=Pagination%20(ui.bootstrap.pagination)

**User Impact:**
Users relying on screen readers cannot understand what the ‘Next’ and ‘Previous’ controls do because their names are not announced. This makes it difficult to navigate pages and may cause users to lose their place or miss important content.

**Pre-Requisites:**
Enable NVDA

**Repro Steps:**

1. Open the URL:[ (https://angular-ui.github.io/bootstrap/#!%23getting_started:~:text=Pagination%20%28ui.bootstrap.pagination%29) ](https://angular-ui.github.io/bootstrap/#!#getting_started:~:text=Pagination%20(ui.bootstrap.pagination))in edge browser in windows.
2. 'Angular directives for bootstrap - Pagination' page will open.
3. Press tab key to navigate to the '>', '>>', '<', '<<' controls.
4. Verify whether screen reader announces the name of 'Next' and 'Previous' controls in the pagination section or not.

**Actual Result:**
Screen reader does not announce the name of 'Next' and 'Previous' controls in the pagination section in windows.
Screen reader announces as 'Link same page'

**Refer Attachment:**

1. Screen reader does not announce the name of 'Next' and 'Previous' controls in the pagination section in windows.png
2. Screen reader does not announce the name of 'Next' and 'Previous' controls in the pagination section in windows.mp4

https://github.com/user-attachments/assets/d844accf-112a-4868-a493-af2553da3320
<img width="969" height="512" alt="Image" src="https://github.com/user-attachments/assets/58f58489-00a7-422c-acdf-62788a0e83d1" />

**Expected Result:**
Screen reader should announce the name of 'Next' and 'Previous' controls in the pagination section in windows.
Screen reader should announce as 'Previous Link same page'

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by AntonChesnokov** [comment]
Hi @msftedad ,
You're mentioning a different project. `angular-ui` is a set of components for legacy Angular (AngularJS). 
This is `ng-bootstrap` created for Angular 2+. Same is true for https://github.com/ng-bootstrap/ng-bootstrap/issues/4893


You can see that on the top of the page that you linked https://angular-ui.github.io/bootstrap/#!#getting_started
